recipes/fans/recipe.yaml

```
@@ -0,0 +1,76 @@
+context:
+  version: "0.3.0"
+
+package:
+  name: fans
+  version: ${{ version }}
+
+source:
+  url: https://github.com/DataAnalyticsEngineering/FANS/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 1573478d9acd27a9cfc126a4167ce3951ec26e9896bf19577b44cbc4bdc48b88
+  patches:
+    - fix_gitignore_ms.patch  # modify so gitignore does not include the sphere32.h5 file
+
+build:
+  number: 0
+  skip:
+    - win
+    - osx
+  script:
+    - if: linux
+      then:
+        - rm -rf build
+        - mkdir build
+        - cd build
+        - cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_C_COMPILER=$PREFIX/bin/mpicc -DCMAKE_CXX_COMPILER=$PREFIX/bin/mpicxx
```

A lot of useful basics comes in via `CMAKE_ARGS`
```suggestion
        - cmake .. $CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_C_COMPILER=$PREFIX/bin/mpicc -DCMAKE_CXX_COMPILER=$PREFIX/bin/mpicxx
```